### PR TITLE
hide api key with env variable

### PR DESCRIPTION
**Pull request by anthonysingiser**
-hide key in .env variable

### CODE REVIEW

File: bellLabs-ui/src/services/SearchRecipe.jsx [hunk 1]
```diff
@@ -15,7 +15,7 @@ const SearchRecipe = () => {
             const response = await axios.get('https://api.spoonacular.com/recipes/complexSearch', {
                 params: {
                     query: query, //
-                    apiKey: '1fd5a4eb995e4292b43a689019516f9b' , // API key
+                    apiKey: import.meta.env.VITE_SPOONACULAR_API_KEY , // API key
```
Comment (Kayboy4140) [inline bellLabs-ui/src/services/SearchRecipe.jsx:18]: Hey @anthonysingiser I see you are trying to hide the API key. Where exactly have you stored the real API key?

**Comment by anthonysingiser** [comment]
you have to set up your own .env file locally, the instructions are in the
group discord chat.

File: bellLabs-ui/src/services/SearchRecipe.jsx (hunk 1 above)
Comment (Kayboy4140) [reply bellLabs-ui/src/services/SearchRecipe.jsx:18]: Got you. Thanks

**Review by Kayboy4140 — APPROVED** [review]
Thanks for this.

**Pull request merged**